# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../ipt_sysrq/fpic.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/Makefile-2.6.orig	2007-07-30 13:47:48.000000000 +0000
+++ ./src/Makefile-2.6	2007-07-30 13:49:19.000000000 +0000
@@ -40,7 +40,7 @@
 	$(MAKE) -C $(KERNEL_SOURCES) SUBDIRS=$(PWD) modules
 
 libipt_SYSRQ.o:	libipt_SYSRQ.c
-	gcc -Wall -DNETFILTER_VERSION=\""$(IPTVERSION)"\" -c $< -o $@
+	gcc -Wall -fPIC -DNETFILTER_VERSION=\""$(IPTVERSION)"\" -c $< -o $@
 
 libipt_SYSRQ.so:	libipt_SYSRQ.o
 	ld -shared -o $@ $<
